test/functional/plugin/lsp_spec.lua

```diff
@@ -3879,6 +3879,73 @@ describe('LSP', function()
         end,
       }
     end)
+
+    it('refresh multiple buffers', function()
```

Comment:
I think the test case could be simplified a bit by using the `_create_server` functionality instead of the `test_rpc_server`. See `Fallback to command execution on resolve error` for an example of how it is used.

Would be great if you could take a look at changing that. But if not I can also merge this as is, as it is already a clear improvement.

Reply:
Thanks for the review!
I'll try to fix it based on your review.

Reply:
@mfussenegger I fixed it.